bpf/lib/nodeport.h

```diff
@@ -1542,6 +1542,9 @@ int tail_nodeport_nat_ipv4(struct __ctx_buff *ctx)
 		}
 	}
 #endif
+	/* Handles reverse SNAT for reply packets from remote backends on ingress
+	 * and SNAT on egress.
+	 */
```

Comment:
This is still not 100% correct, technically, on tc (XDP) both is handled on ingress side.

Reply:
I'm not sure I understand. Do you mean that for XDP, the value of `dir` is independent of ingress/egress since there's only ingress? I was referring to `dir` here, maybe that's what needs to be clarified?

Reply:
Ah yes, I got confused with tc ingress vs egress. It's all handled on ingress, but `dir` can be `NAT_DIR_INGRESS`/`EGRESS`.

Reply:
Ok, I'll clarify that I'm referring to `NAT_DIR_INGRESS` and `NAT_DIR_EGRESS`. That was unclear :-(

Reply:
Small last nit for readability, I would first describe `Handles SNAT on NAT_DIR_EGRESS, and reverse SNAT [...]` given this is more of a logical flow. Rest looks good.